naga/tests/out/spv/atomicCompareExchange-int64.spvasm

```diff
@@ -1,17 +1,17 @@
 ; SPIR-V
 ; Version: 1.0
 ; Generator: rspirv
-; Bound: 125
+; Bound: 123
```

Comment:
Why did this change?

Reply:
Oh I remember why - the old `As` code ended up allocating an `id` that it never used. The new code does not have this property, so there's some small renumberings in the other spvasm files.